packages/perseus-editor/src/item-extras-editor.tsx

```diff
@@ -51,6 +51,13 @@ class ItemExtrasEditor extends React.Component<Props> {
                                     this.props.onChange({
                                         calculator: e.target.checked,
                                     });
+                                    if (!e.target.checked) {
+                                        // When we remove the periodic table,
+                                        // we should also remove sub-options.
+                                        this.props.onChange({
+                                            periodicTableWithKey: false,
+                                        });
+                                    }
```

Comment:
**question**: Why are we doing this when toggling the calculator?

Reply:
Oops-- did I add this twice??